src/components/campaignLandingPage/ResourcesPanel.tsx

```diff
@@ -1,71 +1,75 @@
-                </p>
-
-                {/* TODO: In the Liquid template, this one is NOT a va-link where other links are.
-                 * note to Zach Button to ask the team why */}
-                <a
-                  aria-label={`Download ${resource.entity.name} PDF`}
-                  download
-                  href={resource.entity.fieldMediaExternalFile.uri}
-                  onClick={() => {
-                    // TODO: recordEvent analytics function needs to be implemented
-                    // recordEvent({ 'event': 'clp-link-click', 'clp-section-category': 'Downloadable resources', 'clp-section-title': fieldClpResourcesHeader, 'clp-total-sections': clpTotalSections, 'clp-click-label': `Download ${resource.entity.name}` });
-                  }}
-                >
-                  Download (PDF)
-                </a>
-              </div>
-            </div>
-          ))}
-        </div>
+      <div className="vads-l-row vads-u-margin-bottom--2 medium-screen:vads-u-margin-x--neg1">
+        {resources.documents.map((doc) => (
+          <div
+            key={doc.id}
+            className="vads-l-col--12 medium-screen:vads-l-col--4 vads-u-align-content--stretch vads-u-margin-y--1 "
+          >
+            <div className="vads-u-background-color--gray-light-alt vads-u-height--full vads-u-padding--2 vads-u-display--flex vads-u-flex-direction--column vads-u-justify-content--space-between medium-screen:vads-u-margin-x--1 medium-screen:vads-u-margin-y--0">
+              <h3 className="vads-u-margin--0">{doc.name}</h3>
+              <p className="vads-u-flex--1">{doc.description}</p>
 
-        {placeholders.fieldClpResourcesCta.entity.fieldButtonLink &&
-          placeholders.fieldClpResourcesCta.entity.fieldButtonLabel && (
-            <div className="vads-l-row">
-              <div className="vads-u-col--12">
-                <va-link-action
-                  href={
-                    placeholders.fieldClpResourcesCta.entity.fieldButtonLink.url
-                      .path
-                  }
-                  type="secondary"
-                  text={
-                    placeholders.fieldClpResourcesCta.entity.fieldButtonLabel
-                  }
-                />
-              </div>
+              <a
+                aria-label={`Download ${doc.fileName} PDF`}
+                download
+                data-testid="resource-link"
+                href={doc.url}
+                onClick={() => {
+                  recordEvent({
+                    event: 'clp-link-click',
+                    'clp-section-category': 'Downloadable resources',
+                    'clp-section-title': encodeURIComponent(resources.header),
+                    'clp-total-sections': pageSectionCount,
+                    'clp-click-label': `Download ${encodeURIComponent(doc.name)}`,
+                  })
+                }}
+              >
+                Download (PDF)
+              </a>
```

Comment:
This is the link I mentioned in standup. In the drupal template, it's also not a `<va-link>`, but I wasn't able to find any context on *why* that is. I'm wondering if it's because of the `download` prop or the onClick, but I really don't know.

I propose doing it as-is for now and then figure out whether to change it in a followup ticket (unless someone has context).

Reply:
You know, I feel like I ran into that during https://github.com/department-of-veterans-affairs/next-build/commit/ae30ba105bf15e4b9bbd19164d93aca1fbd279fe, but for some reason my final merge commit still uses `va-link`, and I can't find any notes or comments of mine on the subject. I wonder if my page is broken.